lib/cli/do_recovery/do_activate.js

```diff
@@ -0,0 +1,105 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+/*
+ * Copyright 2019 Joyent, Inc.
+ *
+ * `kbmctl recovery activate ...`
+ */
+'use strict';
+
+const cmdln = require('cmdln');
+
+function do_activate(subcmd, opts, args, cb) {
+    // Given how cmdln instantiates the subcommands, this rule is useless:
+    /* eslint-disable no-invalid-this */
+    if (opts.help) {
+        this.do_help('help', {}, [subcmd], cb);
+        return;
+    } else if (args.length === 0) {
+        cb(new cmdln.UsageError('missing UUID argument'));
+        return;
+    } else if (args.length > 1) {
+        cb(new cmdln.UsageError('incorrect number of arguments'));
+        return;
+    }
+
+    var log = this.top.log;
+    var kbmapi = this.top.kbmapi;
+    const uuid = args[0];
+    var updateOpts = {
+        uuid: uuid,
+        action: 'activate'
+    };
+
+    if (opts.pivtoken) {
+        updateOpts.pivtoken = opts.pivtoken;
+    }
+
+    if (opts.force) {
+        updateOpts.force = opts.force;
+    }
+
+    kbmapi.updateRecoveryConfiguration(updateOpts,
+        function upCb(err, _recCfg, res) {
+        if (err) {
+            log.error({
+                err: err,
+                updateOpts: updateOpts
+            }, 'Update recovery configuration error');
+            cb(err);
+            return;
+        }
+
+        console.log('done');
+        cb();
+    });
+    /* eslint-enable no-invalid-this */
+}
+
+do_activate.options = [
+    {
+        names: ['help', 'h'],
+        type: 'bool',
+        help: 'Show this help.'
+    },
+    {
+        names: ['force', 'f'],
+        type: 'bool',
+        help: 'Force activation (required when pivtoken is given)\n' +
+        'or when there are CNs where the recovery configuration has not\n' +
+        'been staged.'
+    },
+    {
+        names: ['pivtoken', 'p'],
+        helpArg: 'PIVTOKEN',
+        type: 'string',
+        help: 'GUID of the PIVToken to activate (when optionally want to ' +
```

Comment:
`opts.pivtoken` is optional, I'd say.

Reply:
I think this will be more clear:

```
        help: 'When we want to activate a single CN, GUID of the PIVToken ' +
        'associated with that CN'
```

Reply:
Let me know if the following sounds better:
```
        help: 'When we want to activate a single CN, GUID of the PIVToken ' +
        'associated with that CN'
```

Reply:
👍